README.rst

```diff
@@ -121,6 +121,11 @@ Pass the ``-q`` or ``--quick`` command line arguments, or as:
 
     tags = exifread.process_file(f, details=False)
 
+To process makernotes only, without extracting the thumbnail image (if any):
+
+.. code-block:: python
+    tags = exifread.process_file(f, details=True, extract_thumbnail=False)
```

Comment:
Great, almost there! You just need to add a line between the code-block declaration and the sample code.

Otherwise it wont render properly.

Reply:
Uuups! I always get confused with rst syntax. Should be good now. Thanks!

Reply:
thanks!